micropip/install.py

```diff
@@ -0,0 +1,139 @@
+import asyncio
```

Comment:
Thanks, that's a good point. Perhaps we can put all the exposed functions into a single file, e.g. `api.py` or `command.py`?

Reply:
Well I think we don't want people to import everything from micropip.api or micropip.command either. 

IMO we would make public here mostly top level functions (install/uninstall, freeze etc). So those would be importable from `micropip` directly via the top level `__init__`, all the actual modules where the code is could probably be private. 

I mean this is exactly what we had before with `micropip/_mock_package.py` and `micropip/_micropip.py`. So if we split those files into other files, it would make sense to keep the same logic IMO.

Reply:
Yeah, I mean we will definitely not change the top-level functions. I was thinking to put public APIs into a single file or directory to prevent confusion like `from micropip import install` you mentioned.

Reply:
I moved public APIs into `commands` subdirectory (similar to [pip](https://github.com/pypa/pip/tree/main/src/pip/_internal/commands)), and let `__init__.py` import from it.